GitCommands/CommitTemplateManager.cs

```diff
@@ -63,7 +76,16 @@ public CommitTemplateManager(IGitModule module)
         /// <summary>
         /// Gets the collection of all currently registered commit templates provided by plugins.
         /// </summary>
-        public IEnumerable<CommitTemplateItem> RegisteredTemplates => RegisteredTemplatesDic.Select(item => new CommitTemplateItem(item.Key, item.Value()));
```

Comment:
Why couldn't you have just added something like `.OrderBy(template => template.Name)` here?

Reply:
We shouldn't hardcode sorting algorithm because users can use custom JQL sorting. Users can sort Jira issues by date, key or by other attributes. Therefore I choose this approach (to save initial ordering of Jira query result) to solve this problem.